src/components/NumericStepper/NumericStepper.jsx

```diff
@@ -0,0 +1,330 @@
+import React, {PropTypes, Component} from 'react';
+import {themr} from 'react-css-themr';
+import {DEBOUNCE} from 'dx-util/src/function/function.js';
+import {PURE} from 'dx-util/src/react/pure';
+import Input from '../Input/Input.jsx';
+import ButtonIcon from '../ButtonIcon/ButtonIcon.jsx';
+import Button from '../Button/Button.jsx';
+import classnames from 'classnames';
+
+export const NUMERIC_STEPPER = Symbol('NumericStepper');
+
+const BUTTON_TYPE = {
+	UP: 'up',
+	DOWN: 'down'
+};
+
+const KEYCODE = {
+	UP: 38,
+	DOWN: 40
+};
+
+@PURE
+@themr(NUMERIC_STEPPER)
+export default class NumericStepper extends Component {
+
+	static propTypes = {
+		...Input.propTypes,
+		step: React.PropTypes.number,
+		min: React.PropTypes.number,
+		max: React.PropTypes.number,
+		formatter: React.PropTypes.func,
+		ButtonComponent: React.PropTypes.func,
+		InputComponent: React.PropTypes.func,
+
+		repeatIntervar: React.PropTypes.number,
+		repeatDelay: React.PropTypes.number,
```

Comment:
Then they should probably be not. Instead it's better to have an ability to pass down custom Holdable